src/plugin/asyncMessageHandlers/initiate.ts

```diff
@@ -41,7 +41,7 @@ export const initiate: AsyncMessageChannelHandlers[AsyncMessageTypes.INITIATE] =
     const oldTokens = await getTokenData();
     if (oldTokens) {
       notifyTokenValues({
-        ...oldTokens, activeTheme, usedTokenSet, storageType,
+        ...oldTokens, activeTheme, usedTokenSet, storageType, userId: currentUser?.id, licenseKey,
```

Comment:
can we separate these two value groups rather like this

```suggestion
        values: { ...oldTokens, activeTheme, usedTokenSet, storageType },
        userData: { userId: currentUser?.id, licenseKey }
```

and then change notifyTokenValues to accept these like that? and add useData as its own type? that way we're separating this we store on the document with user data we submit